lib/http/request.ts

```diff
@@ -69,7 +76,13 @@ export function httpRequest(sdk: OktaAuthHttpInterface, options: RequestOptions)
       if (res && isString(res)) {
         res = JSON.parse(res);
         if (res && typeof res === 'object' && !res.headers) {
-          res.headers = resp.headers;
+          if (Array.isArray(res)) {
```

Comment:
Which response is returning an array at the top-level?

Reply:
`/idp/myaccount/emails` and `/idp/myaccount/phones`